js/store.js

```diff
@@ -360,6 +363,29 @@ const store = {
     // empty out request processing queue
     capture.queue = [];
     return await this.db.websites.clear();
+  },
+
+  async getOldestDate() {
+    const oldestSite = await this.db.websites.orderBy('requestTime').first();
+    if (!oldestSite) {
+      return '';
+    }
+    return oldestSite.requestTime;
+  },
+
+  async getNumSitesVisited() {
+    // Dexie does not accept boolean values; using 0/1 instead
```

Comment:
Change this comment to be indexeddb indexes can't be boolean. Perhaps put it in one place rather than copy it in many?